### PR TITLE
Exclude LGPL licensed dependencies

### PR DESCRIPTION
**Pull request by prb112**
Signed-off-by: Paul Bastide <pbastide@us.ibm.com>

### CODE REVIEW

**Review by lmsurpre — COMMENTED** [review]
does it break any of the cql or term-graph stuff?

**Comment by lmsurpre** [comment]
At least the CQL stuff has some tests...so hopefully everything green means its good-to-merge.  Will approve but lets wait for @csandersdev 's approval as well.

**Review by lmsurpre — APPROVED** [review]

**Comment by csandersdev** [comment]
The dependency being excluded from the CQL modules is a transitive dependency from some JAXB code (org.jvnet.jaxb2_commons). I'm not really clear on how the javaparser LGPL code is used in that module, so I'm going to have to take some guesses. There are a few areas where I think a JAXB issue might manifest - parsing ELM attachments of libraries, reading a model info file for CQL translation, and then writing the translated ELM back out. All of those things are exercised by the unit tests / server tests which are still passing after this change. I think this is ok for now.

**Pull request merged**
